src/OperationProcessor.ts

```diff
@@ -56,6 +70,14 @@ class OperationProcessorImpl implements OperationProcessor {
     return this.operationStore.put(operation);
   }
 
+  /**
```

Comment:
`process` method should be removed. This also raises the question that since there is only one implementation of the `OperationProcessor` interface, maybe there is no need for that interface.

Reply:
Yes, makes sense.

Reply:
Took the suggestion.